client/src/app/auth/login.tsx

```diff
@@ -1,7 +1,52 @@
+import { ChangeEvent, FormEvent, useState } from "react";
+import Button from "../components/button";
+import TextField from "../components/text-field";
+
 export default function Login() {
   return (
     <div>
       <h1>Log in</h1>
+      <ExecLogIn />
     </div>
   );
 }
+
+
+function ExecLogIn() {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  function handleInputChange(e: ChangeEvent<HTMLInputElement>): void {
+    const { name, value } = e.target;
+    if (name === 'username') {
+      setUsername(value);
+    } else if (name === 'password') {
+      setPassword(value);
+    }
+  }
+
+  function handleLogin(e: FormEvent<Element>): void {
+    throw new Error("Function not implemented.");
+    //call the User Authenitcation API
+  }
+
+  return(
+    <div>
+    <form>
+        <TextField
+          label="Username"
+          name="username"
+          value={username}
+          onChange={handleInputChange}
+        />
+        <TextField //todo change the component to obfuscate the text
+          label="Password"
+          name="password"
+          value={password}
+          onChange={handleInputChange}
+        />
```

Comment:
You don't have to do this now but if you like, you can add a prob to the `TextField` component to specify that its a password field. You'll have to edit the actual `TextField` component for this.